Update ikea.py

Philipameen opened the pull request:
Updated commands to match changes made to the zha quirk for E1743.
Changes have been made for Hold "O" and Release "I" or "O".

xaviml commented:
Thank you @Philipameen for the PR. If I understood correctly from the linked issue #455, other events should also be fixed (like the clicks). Did you check other events?

Philipameen commented:
I checked all events and for the E1743 only these two had been changed.
I only have the E1743 so I don't know about the changes made to the five button controller.
According to this pull in the latest release probably more controllers are affected by the change to the stop command.
https://github.com/zigpy/zha-device-handlers/pull/1465

xaviml commented:
Hi @Philipameen ,

Could you check if 2022.4.2 has fixed this issue? (with ControllerX v4.18.2). According to this post, the issue has been solved:

https://github.com/xaviml/controllerx/issues/455#issuecomment-1094972607

Regards,

Philipameen commented:
Hello, I updated to 2022.4.2 (from 2022.4.1) and have ControllerX v4.18.2.
Checked all events and for me they are still the same. Latest update seemed to have fixed it for others so maybe it's just me? For me it's not a problem, I just changed the mapping.

xaviml commented:
Hi @Philipameen,

I have added these changes in #459 together with the fix of other IKEA ZHA mappings.

Regards,
Xavi M.
